KeyboardHook.cs

```diff
@@ -40,7 +41,12 @@ private IntPtr HookCallback(int nCode, IntPtr wParam, IntPtr lParam)
                     try
                     {
                         System.Threading.Thread.Sleep(killerSpeed);
-                        Process.Start(GameLauncherPath);
+                        Process proc = Process.Start(GameLauncherPath);
+                        if (PriorityAffinity)
+                        {
+                            proc.ProcessorAffinity = (System.IntPtr)0x08;

```

Comment:
Why 0x08? Is this code assuming that there are at least 4 cores? Doesn't 0x1 seem like a better choice?

Reply:
Yeah I thought about it as well, but rl_evader's bat file used 8 so I thought I will follow it to be the same for safety, I will ask him if it matters

Reply:
He replied saying 4th core was his personal preference as it seemed to work better for ledge storage. But it makes sense to make it 0x1, I will change

Reply:
Resolved by 9e8a5c2d27137ca5f310cb2978063eb50d7508fd